translations/spanish/main.ts

```diff
@@ -128,15 +128,15 @@ export const translations: Translations = {
 		"Your username contains a phrase banned by this room.": "Tu nombre de usuario contiene una frase prohíbida en esta sala.",
 		"Your status message contains a phrase banned by this room.": "Tu status contiene una frase prohíbida en esta sala.",
 
-		"You are ${lockType} and can only private message members of the global moderation team. ${lockExpiration}": "",
-		"Get help with this": "",
-		"The user \"${targetUser.name}\" is locked and cannot be PMed.": "",
+		"You are ${lockType} and can only private message members of the global moderation team. ${lockExpiration}": "Estás ${lockType} y solo puedes enviar mensajes privados al equipo de modreación global. ${lockExpiration}",
+		"Get help with this": "Consigue ayuda con esto",
+		"The user \"${targetUser.name}\" is locked and cannot be PMed.": "El usuario \"${targetUser.name}\" tiene un lock y no puede recibir mensajes privados.",
 		"On this server, you must be of rank ${groupName} or higher to PM users.": "",
-		"This user is blocking private messages right now.": "",
-		"This ${Config.groups[targetUser.group].name} is too busy to answer private messages right now. Please contact a different staff member.": "",
-		"If you need help, try opening a <a href=\"view-help-request\" class=\"button\">help ticket</a>": "",
-		"You are blocking private messages right now.": "",
-		"You are blocking challenges right now.": "",
+		"This user is blocking private messages right now.": "Este usuario está bloqueando mensajes privados por el momento",
```

Comment:
```suggestion
		"This user is blocking private messages right now.": "Este usuario está bloqueando mensajes privados por el momento.",
```
Please be consistent with end punctuation; I don't believe this is an actual grammatical difference with Spanish (if it is, then I stand corrected).